src/praisonai-agents/praisonaiagents/tools/skill_tools.py

```
@@ -16,22 +16,33 @@
 
 
 class SkillTools:
-    """Tools for executing Agent Skills scripts."""
+    """Tools for executing Agent Skills scripts and managing skills."""
     
-    def __init__(self, working_directory: str = None):
+    def __init__(self, working_directory: str = None, workspace=None):
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)